db/migrate/data/20160127210624_convert_configurations_to_settings_changes/vmdb.tmpl.yml

```diff
@@ -332,6 +332,8 @@ workers:
         :amqp_heartbeat: 30
       :event_catcher_amazon:
         :poll: 15.seconds
+      :event_catcher_hawkular:
+        :poll: 15.seconds
```

Comment:
ok, just out of curiosity: is this an exact copy of the log or a grep?
My guess is, that the [monitor thread](https://github.com/durandom/manageiq/blob/master/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L154) exits. Because [this](https://github.com/durandom/manageiq/blob/master/app/models/manageiq/providers/base_manager/event_catcher/runner.rb#L112) returns